fbw-a32nx/src/systems/instruments/src/SD/Pages/Crz/Crz.tsx

```diff
@@ -213,9 +214,16 @@ export const PressureComponent = () => {
 };
 
 export const CondComponent = () => {
-    const [cockpitCabinTemp] = useSimVar('L:A32NX_COND_CKPT_TEMP', 'celsius', 1000);
-    const [fwdCabinTemp] = useSimVar('L:A32NX_COND_FWD_TEMP', 'celsius', 1000);
-    const [aftCabinTemp] = useSimVar('L:A32NX_COND_AFT_TEMP', 'celsius', 1000);
+    const [unit] = usePersistentProperty('CONFIG_USING_METRIC_UNIT', '1');
+    let [cockpitCabinTemp] = useSimVar('L:A32NX_COND_CKPT_TEMP', 'celsius', 1000);
+    let [fwdCabinTemp] = useSimVar('L:A32NX_COND_CKPT_TEMP', 'celsius', 1000);
```

Comment:
why fwd and aft cabin get the temp from the same lvar `L:A32NX_COND_CKPT_TEMP`.

this make all temps become the same value